lib/pronto/formatter/github_status_formatter.rb

```diff
@@ -0,0 +1,85 @@
+require_relative 'github_status_formatter/sentence'
+
+module Pronto
+  module Formatter
+    class GithubStatusFormatter
+      def initialize(opts = {})
+        @level_mapping ||= opts.fetch(:level_mapping, {})
```

Comment:
Not sure if it's worth having a way to override it. Let's leave it off for now 😄